### ISSUE TITLE
Capabilities override

### ISSUE THREAD

**Issue by alcarney** [swyddfa/lsp-devtools#98]
Provide a cli option to override the set of capabilities a set of tests are run with.

**Comment by alcarney** [comment]
Don't think we need to add anything special to support this, but adding some notes to the docs would be a good idea

**Comment by DetachHead** [comment]
it would be nice if `lsp-devtools client` also had an option to specify its capabilities. currently it just sends `{}`

**Comment by alcarney** [comment]
> it would be nice if lsp-devtools client also had an option to specify its capabilities. 

I agree, the client is just a proof of concept at this stage but I definitely want to get around to doing it "properly" at some point
